keyboards/anavi/arrows/info.json

```
@@ -0,0 +1,72 @@
+{
+    "keyboard_name": "arrows",
+    "manufacturer": "ANAVI",
+    "url": "https://github.com/AnaviTechnology/anavi-arrows",
+    "maintainer": "leon-anavi",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "matrix_pins": {
+        "direct": [
+            ["GP27", "GP28", "GP29", "GP0", "GP4"]
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "backlight": true,
+        "oled": true,
+	"combo": true
```

This cannot be enabled at the keyboard level.
```suggestion
        "oled": true
```

---

OK, thank you. I will move it to `keyboards/anavi/arrows/keymaps/default/rules.mk`